File: src/main/java/com/natia/secretmod/networking/ColorCosmetics.java
package com.natia.secretmod.networking;

import static com.natia.secretmod.utils.WebUtils.fetch;

public class ColorCosmetics {
    public ColorCosmetics(String targetColor) {
        fetch("https://secretmod-hypixel.herokuapp.com/colors?color=" + targetColor, res -> {
            if (!res.asString().equals("-1")) {
                String colors = res.asJson().get("color").getAsString();
                colors = colors.replace("[(<splitter>)", "\n");
                users = colors;
            }
        });
    }

    public String users;
}

File: src/main/kotlin/natia/skytweaks/networking/ColorCosmetics.kt
package natia.skytweaks.networking

import natia.skytweaks.utils.WebUtils.fetch

class ColorCosmetics(targetColor: String) {

    var users: String = ""

    init {
        fetch("https://skytweaks-web.vercel.app/colors?color=$targetColor") { res ->
            if (res.asString() != "-1") {
                var colors = res.asJson().get("color").asString
                colors = colors.replace("[(<splitter>)", "\n")
                users = colors
            }
        }
    }
}
